source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
```

Comment:
@ltran-mdb2 I think this might best for this to live in it's own nav as it is a complimentary feature to "Export to Language". We could call it "Test Application Query" in the nav and make the title "Export and Test Application Query in Playgrounds".

Lastly, on both of them (i.e. Export to Language) - could we call out that these features are currently only available for users who have Github Copilot & MongoDB's Copilot extension enabled?